<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 68, 68);
color: rgb(212, 212, 212);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="SettingsTabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
	border: 2px solid #C2C7CB;
}
QTabWidget::tab-bar {
	left: 5px;
}
QTabBar::tab {
	background: rgb(68, 68, 68);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {
	background: rgb(95, 95, 95);
}
QTabBar::tab:selected {
	border-color: #9B9B9B;
	border-bottom-color: #C2C7CB;
}
QTabBar::tab:!selected {
	margin-top: 2px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralSettingsTab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QLineEdit" name="AnimeFolderLineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>125</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="AnimeFolderBrowse">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
      <widget class="QLabel" name="AnimeFolderSettingsLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Anime Folder:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="SubtitleLanguageLineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>125</width>
         <height>23</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QLabel" name="SubtitleLanguageSettingsLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Subtitle Language:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="IgnoreSongsSignsCheckbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>223</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ignore Songs and Signs Subtitles</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="AnilistSettingsTab">
      <attribute name="title">
       <string>Anilist</string>
      </attribute>
      <widget class="QLabel" name="AnilistInstructionsLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>395</width>
         <height>69</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;To connect a new account, click Connect to anilist to the left&lt;/p&gt;&lt;p&gt;Login, then copy the code provided, enter it in the text box below&lt;/p&gt;&lt;p&gt;Then hit Confirm Code to the right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="AnilistConnectedAccountLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>147</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Connected account: N/A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="AnilistConnect">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>114</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Connect to anilist</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AnilistCodeConfirm">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>30</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirm Code</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="AnilistCodeBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>411</width>
         <height>130</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
